HW13/ClassWork1.py

```diff
@@ -0,0 +1,13 @@
+string = '()0)aa()'
+string_new = ''
+i = 0
+
+for i in range(len(string)):
+    b = i+1
+    if string[i] == '(' and string[b] == ')':
```

Comment:
You could have just used `if string[i] == '(' and string[i+1] == ')':` here and therefore wouldn't have needed a separate variable `b` at all.